src/TLuaInterpreter.cpp

```diff
@@ -12248,6 +12249,19 @@ void TLuaInterpreter::initLuaGlobals()
     luaopen_yajl(pGlobalLua);
     lua_setglobal(pGlobalLua, "yajl");
 
+    // prepend profile path to package.path and package.cpath
+    // with a singleShot Timer to avoid crash on startup.
+    // crash caused by calling Host::getName() too early.
+    QTimer::singleShot(0, [this]() {
+        QChar separator = QDir::separator();
+
+        luaL_dostring(pGlobalLua, QStringLiteral("package.path = getMudletHomeDir() .. [[%1?%1init.lua;]] .. package.path").arg(separator).toUtf8().constData());
+        luaL_dostring(pGlobalLua, QStringLiteral("package.path = getMudletHomeDir() .. [[%1?.lua;]] .. package.path").arg(separator).toUtf8().constData());
+
+        luaL_dostring(pGlobalLua, QStringLiteral("package.cpath = getMudletHomeDir() .. [[%1?;]] .. package.cpath").arg(separator).toUtf8().constData());
```

Comment:
The `package.cpath` should entry should also contain the file name ending for shared libraries on the platforms. These are `.so` for *nix, `.dylib` for mac and `.dll` for windows.

Reply:
__Cygwin__ really confuses the Windows/Linux devision BTW...! 😝

Reply:
The reason behind this is clarity. I'm not very familiar with lua outside the mudlet context, so please, let me know if this is something you want.

I don't think that hiding crossplatform managment to developers is a good idea and I'd rather have them using a less clean and automated requires if this mean not favouring platform dependant development. 

In short, by not assuming the file extension in cpath, the developer has to be aware of it.

Then again, it's not that big of a deal in my opinion so just let me know how you think this is best.